### PR TITLE
new files, break up footer

### PR DESCRIPTION
**Pull request by rgoodie**
Created new files in which to break up the footer a bit.

### CODE REVIEW

File: src/components/footer.jsx [hunk 1]
```diff
@@ -1,5 +1,9 @@
 import React from 'react';
 import { Link } from 'react-router-dom';
+import FooterLogin from './footer/login';
+import FooterGuidance from './footer/guidance'
+import FooterConnect from './footer/connect'
```
[the remark this answers is not in the document]
Comment (rgoodie) [reply src/components/footer.jsx]: done

File: src/components/footer.jsx [hunk 2]
```diff
@@ -12,37 +16,13 @@ const Footer = () => (
...
-                <br/>
-                <a href="#" className="text-dark">
-                  Log in
-                </a>
-              </p>
+              <FooterLogin/>
+              
             </div>
           </div>
           {/* footer-guidance */}
```
[the remark this answers is not in the document]
Comment (rgoodie) [reply src/components/footer.jsx]: removed
```
$ git log --oneline 
32bb42a removed footer login
7c09bf2 changed to Social"
```

File: src/components/footer/guidance.jsx [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+import React from 'react';
+import { Link } from 'react-router-dom';
+
+const FooterGuidance = () => (
+
+    <div className="col-md-2 tracked">
+        <div className="float-md-right text-md-left">
+            <h6 className="footer-type text-uppercase text-secondary">
+                Guidance
+   </h6>
```
[the remark this answers is not in the document]
Comment (rgoodie) [reply src/components/footer/guidance.jsx]: Think that was due to a document formatter I might have used. Cleaned up a bit

File: src/components/footer.jsx [hunk 4]
```diff
@@ -11,89 +15,27 @@ const Footer = () => (
...
-                </span>
-                <br/>
-                <a href="#" className="text-dark">
-                  Log in
-                </a>
-              </p>
+                         
             </div>
           </div>
           {/* footer-guidance */}
```
Comment (meltedspork) [inline src/components/footer.jsx]: we don't need `{/* footer-guidance */}` - we can remove that

File: src/components/footer.jsx [hunk 5]
```diff
@@ -11,89 +15,27 @@ const Footer = () => (
...
-                  <a href="#" className="text-dark">
-                    Blog
-                  </a>
-                </li>
-              </ul>
-            </div>
-          </div>
+          <FooterGuidance/>
+
+          {/* footer-about us */}
```
Comment (meltedspork) [inline src/components/footer.jsx]: we don't need `{/* footer-about us */}` - we can remove that

File: src/components/footer.jsx [hunk 6]
```diff
@@ -11,89 +15,27 @@ const Footer = () => (
...
-            <div className="col-md-6">
-              <div className="text-white footer-type tracked text-uppercase float-md-left">
-                <p>
-                  Built with <i className="fa fa-heart" aria-hidden="true" aira-label="heart"></i> in Chicago
-                </p>
-              </div>
-            </div>
+            <FooterChicagoBanner/>  
+            
             {/* footer-banner-social */}
```
Comment (meltedspork) [inline src/components/footer.jsx]: we don't need `{/* footer-banner-social */}` - we can remove that

File: src/components/footer/chicago-banner.jsx [hunk 7]
```diff
@@ -0,0 +1,12 @@
...
+import { Link } from 'react-router-dom';
+const FooterChicagoBanner = () => (
+    <div className="col-md-6">
+        <div className="text-white footer-type tracked text-uppercase float-md-left">
+            <p>
+                Built with <i className="fa fa-heart" aria-hidden="true" aira-label="heart"></i> in Chicago
+                </p>
+        </div>
+    </div>
+)
```
Comment (meltedspork) [inline src/components/footer/chicago-banner.jsx]: add new line between here

File: tests/components/footer.test.js [hunk 8]
```diff
@@ -0,0 +1,17 @@
...
+    let wrapper;
+
+    beforeEach(() => {
+        wrapper = shallow(<Footer />);
+    });
+
+    it ('Footer tag exists', () => {
+        expect(wrapper.find('footer').length).toBe(1)    
+    });
+
```
Comment (meltedspork) [inline tests/components/footer.test.js]: we need add check for footer smaller components here.

**Pull request merged**